frontend/src/components/session/login-form.js

```diff
@@ -160,6 +165,34 @@ class LoginForm extends Component {
             id='password'
           />
         </div>
+        { type === 'signup' && (
+          <div className={ classes.margins }>
+            <TextField
+              error={ this.state.password !== this.state.confirmPassword }
+              helperText={ this.state.password !== this.state.confirmPassword ? 'Passwords do not match!' : '' }
```

Comment:
Great @nguyenquangv97, but we need here on the validation text to handle the translation, we use ReactIntl: https://github.com/formatjs/react-intl

Check out other parts of the application examples of how to use it.